tests/patmat/i6490.check

```diff
@@ -0,0 +1 @@
+8: Pattern Match Exhaustivity: Foo(Nil)
```

Comment:
Shouldn’t it be `Pattern Match Exhaustivity: Foo()`?

Reply:
Now it prints `Foo()`.